tests/unit/test_linear_learner.py

```diff
@@ -222,10 +222,9 @@ def test_num_classes_can_be_string_for_multiclass_classifier(sagemaker_session):
 
 @pytest.mark.parametrize("iterable_hyper_parameters, value", [("max_iterations", [0])])
 def test_iterable_hyper_parameters_type(sagemaker_session, iterable_hyper_parameters, value):
-    with pytest.raises(TypeError):
```

Comment:
reason for `[DRAFT]` status -- this. i hadn't had time to deep dive. i stepped into pdb, but couldn't figure out why it would be passing and had to time box the deep dive at this point.

Reply:
Looking into this.

Reply:
Discussed offline; it looks like this test may be parameterized with the wrong iterable hyperparameter.

Reply:
i've fixed the existing bad test by using `epochs`